### PR TITLE
Add guide to running Federalist locally

### PR DESCRIPTION
**Pull request by jmhooper**
This commit adds a guide which provides some notes about running the different parts of Federalist locally in a development context.

### CODE REVIEW

File: pages/how-federalist-works/running-federalist-locally.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: Running Federalist locally
+parent: How Federalist Works
+---
+
+## Dependencies / Tooling
+
+Before you start you'll want to make sure you have the following installed:
```
Comment (wslack) [inline pages/how-federalist-works/running-federalist-locally.md]: style thing: no need to say "you'll want to" - instead, try "Before you start, ensure you have the following installed:"

File: pages/how-federalist-works/running-federalist-locally.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+See [How Federalist Works]({{site.baseurl}}/pages/how-federalist-works) for a list of the code repositories and descriptions of the different parts of the Federalist system. You'll want to pull the repos relevant to the parts of the system you are working with locally.
+
+## Running federalist
+
+The main Federalist app's code lives in the [18F/federalist](https://github.com/18F/federalist) repository on GitHub. After pulling the app, the `config/local.js` file should be created from the template at `config/local.sample.js`.
+
+In order to run the app needs an S3 bucket and a SQS queue to communicate with. In development we use the staging S3 bucket and the staging SQS queue for this purpose. To this end you can edit your `config/local.js` to contain the credentials of the staging instances of those services.
+
+The federalist app uses GitHub for authentication. You will need to create a [GitHub OAuth application](https://github.com/settings/applications/new) and add the client ID, client secret, and callback URL to your `config/local.js`.
```
Comment (wslack) [inline pages/how-federalist-works/running-federalist-locally.md]: One more place for caps

Comment (jmhooper) [inline pages/how-federalist-works/running-federalist-locally.md]: caps?

Comment (wslack) [inline pages/how-federalist-works/running-federalist-locally.md]: "the federalist app" -> Federalist should be capitalized.

**Comment by jmhooper** [comment]
Pushed an amended commit with fixes.

**Pull request merged**